Formula/winetricks.rb

```diff
@@ -11,6 +11,7 @@ class Winetricks < Formula
   depends_on "p7zip"
   depends_on "unrar"
   depends_on "wine"
+  depends_on "zenity" => :optional, "zenity is need for winetricks' GUI"
```

Comment:
This isn't working the way you think it is. If you want to add a comment to an option you need to use the:

```ruby
option "with-zenity", "Needed for GUI"
```
style syntax.